### PR TITLE
CI: skip cli-suite on Windows due to #9571

### PR DESCRIPTION
**Pull request by ulysses4ever**
Further investigation of #9571 is needed, but we can take a breathe in the meantime.

**Template B: This PR does not modify behaviour or interface**

*E.g. the PR only touches documentation or tests, does refactorings, etc.*

Include the following checklist in your PR:

* [x] Patches conform to the [coding conventions](https://github.com/haskell/cabal/blob/master/CONTRIBUTING.md#other-conventions).
* [ ] Is this a PR that fixes CI? If so, it will need to be backported to older cabal release branches (ask maintainers for directions).

### CODE REVIEW

**Comment by ulysses4ever** [comment]
If no one objects, I propose to merge it skipping the 2-day delay. Once we get the two approvals of course.

File: .github/workflows/validate.yml
Comment (geekosaur) [inline .github/workflows/validate.yml:213]: See line 177 for how to do this properly; I think that won't work as is (`runner.os` won't be expanded).

**Review by geekosaur — CHANGES_REQUESTED** [review]

File: .github/workflows/validate.yml
Comment (geekosaur) [reply .github/workflows/validate.yml:213]: Okay, actually it looks like it did work (skipped `cli-suite` only on Windows) but we probably want to be consistent.

Comment (ulysses4ever) [reply .github/workflows/validate.yml:213]: Line 177 is a part of a bash expression, and I act on the yaml level. So, I'm not sure why they should be consistent necessarily: they're pretty different notations...

**Review by geekosaur — APPROVED** [review]

**Comment by geekosaur** [comment]
Do we want to backport this / will this affect CI on 3.12 branch?

**Comment by geekosaur** [comment]
@mergify backport 3.12

**Pull request merged**
